packages/lesswrong/components/users/EAUserTooltipContent.tsx

```diff
@@ -79,7 +79,7 @@ const formatBio = (bio?: string): string => htmlToTextDefault(bio ?? "");
 const formatStat = (value?: number): string => {
   value ??= 0;
   return value > 10000
-    ? `${Math.floor(value / 1000)} ${value % 1000}`
+    ? `${Math.floor(value / 1000)} ${String(value % 1000).padStart(3, "0")}`
```

Comment:
Is there a reason we don't use `toLocalString()`? Commas instead of spaces seems fine to me

Reply:
Having spaces was an explicit part of the figma design, and I kind of like it.

Although, now you say that, it's probably better to just calculate it like `new Intl.NumberFormat("en-GB").format(value).replace(/,/g, " ")`. There's more to go wrong though from a browser-compatability point of view, so I'm probably just going to leave it as-is (we won't have a problem until someone reaches 1,000,000 karma, and if someone ever does that then they probably deserve the notoriety of breaking the site a bit).